lib/list.g

```diff
@@ -875,3 +875,107 @@ DeclareSynonym( "IntersectBlist", INTER_BLIST );
 ##  <#/GAPDoc>
 ##
 DeclareSynonym( "SubtractBlist", SUBTR_BLIST );
+
+#############################################################################
+##
+#F  FlipBlist( <blist> )
+##
+##  <#GAPDoc Label="FlipBlist">
+##  <ManSection>
+##  <Func Name="FlipBlist" Arg='blist'/>
+##
+##  <Description>
+##  Changes every entry in <A>blist</A> that equals <K>true</K> to <K>false</K>
+##  and vice versa. If <C>blist1</C> and <C>blist2</C> are boolean list with
+##  equal length and every value in <C>blist2</C> is <K>true</K>,
+##  then <C>FlipBlist( blist1 )</C> is equivalent to
+##  <C>SubtractBlist( blist2, blist1 ); blist1 := blist2;</C>
+##  but <C>FlipBlist</C> is faster, and simpler to type.
+##  <P/>
+##  <Ref Func="FlipBlist"/> returns nothing, it is only called to change
+##  <A>blist</A> in-place.
+##  <P/>
+##  <Example><![CDATA[
+##  gap> blist1 := [ true, true, true, true ];;
+##  gap> blist2 := [ true, false, true, false ];;
+##  gap> SubtractBlist( blist1, blist2 );
+##  gap> blist1;
+##  [ false, true, false, true ]
+##  gap> FlipBlist( blist1 );
+## [ true, false, true, false ]
+##  ]]></Example>
+##  </Description>
+##  </ManSection>
+##  <#/GAPDoc>
+##
+DeclareSynonym( "FlipBlist", FLIP_BLIST );
+
+#############################################################################
+##
+#F  SetAllBitsBlist( <blist> )
```

Comment:
As Chris has pointed out, this documentation and synonym declaration should be done all in terms of `ClearAllBits`.